src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/custom.py

```diff
@@ -124,64 +125,74 @@ def acr_show(registry_name, resource_group_name=None):
 
     return client.get_properties(resource_group_name, registry_name)
 
-def acr_update(registry_name, #pylint: disable=too-many-arguments
-               resource_group_name=None,
-               storage_account_name=None,
-               tags=None,
-               enable_admin=False,
-               disable_admin=False):
-    '''Update a container registry.
```

Comment:
Unless you have this in your _help.py file, you will want to keep at least this part of the docstring to supply the short and long summary. If it is already in _help.py, then that's not necessary.